python/date_module.py

```diff
@@ -0,0 +1,87 @@
+import mgp
+import pytz
+import datetime
+
+MINUTES_IN_HOUR = 60
+SECONDS_IN_MINUTE = 60
+MILLISECONDS_IN_SECOND = 1000
+HOURS_IN_DAY = 24
+UNIX_EPOCH = datetime.datetime(1970, 1, 1, 0, 0, 0)
```

Comment:
For these constants we often make class Constants into which we store all of them, check tgn.py for example. Please adapt code to follow same style. In general I think this is not mistake, just to follow same code style :)